tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -771,7 +771,7 @@ private void createWildFlyScript() throws Exception {
                 String.format("/subsystem=elytron/filesystem-realm=%s:add(path=%s, levels=%s, credential-store=%s, secret-key=%s)", fileSystemRealmName, fullOutputPath+'/'+fileSystemRealmName, levels, "mycredstore"+counter, secretKeyAlias)
             );
 
-            if (!createScriptCheck.equals("y") && !createScriptCheck.equals("yes")) {
+            if (createScriptCheck.isEmpty() || createScriptCheck.toLowerCase().startsWith("y")) { // Create a new script file, or overwrite the existing one
```

Comment:
Hello @cam-rod , this means if user does not enter neither `n` or `y` , we override the existing file? I think it'd be better for the default behaviour to be to not override?

Reply:
Oh you're right. I had written it like this since the prompt is only shown if a file would be overwritten (the `APPEND` option alone can't create a new file). I'll change the prompt to default to no, like this:

```java
  if (Paths.get(String.format("%s.cli", fileSystemRealmName)).toFile().exists()) {
                createScriptCheck = prompt(
                        false,
                        null,
                        false,
                        ElytronToolMessages.msg.shouldFileBeOverwritten(String.format("%s.cli", fileSystemRealmName))
                );
                if (createScriptCheck.isBlank()) createScriptCheck = "n";
            }